workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs

```diff
@@ -179,11 +200,16 @@ public static LocatorConfig CreateConnectionConfigFromCommandLine(Dictionary<str
 
             var steamDeploymentTag = CommandLineUtility.GetCommandLineValue(
                 parsedArgs, RuntimeConfigNames.SteamDeploymentTag, string.Empty);
+            if (!string.IsNullOrEmpty(steamDeploymentTag))
+            {
+                config.SetSteamDeploymentTag(steamDeploymentTag);
+            }
+
             var steamTicket = CommandLineUtility.GetCommandLineValue(
                 parsedArgs, RuntimeConfigNames.SteamTicket, string.Empty);
             if (!string.IsNullOrEmpty(steamDeploymentTag) && !string.IsNullOrEmpty(steamTicket))
```

Comment:
Looks like we may need to remove the check for `steamDeploymentTag` here

Reply:
Needs a test for when there's only steam ticket and no deployment tag

Reply:
Oh shoot. Thanks for spotting.